Uchu.World/Systems/Behaviors/DamageAbsorptionBehavior.cs

```diff
@@ -1,9 +1,29 @@
+using System;
 using System.Threading.Tasks;
 
 namespace Uchu.World.Systems.Behaviors
 {
     public class DamageAbsorptionBehavior : BehaviorBase
     {
         public override BehaviorTemplateId Id => BehaviorTemplateId.DamageAbsorption;
+
+        private uint AbsorbAmount { get; set; }
+
+        public override async Task BuildAsync()
+        {
+            AbsorbAmount = await GetParameter<uint>("absorb_amount");
+        }
+        public override void ExecuteStart(BehaviorExecutionParameters parameters)
+        {
+            DestroyableComponent absorb;
+            if (parameters.BranchContext.Target != null && parameters.BranchContext.Target.TryGetComponent<DestroyableComponent>(out absorb)) {} 
+            else if (!(parameters.Context.Associate.TryGetComponent<DestroyableComponent>(out absorb))) return;
```

Comment:
This looks messy. Can `out absorb` be changed to `out var absorb` and the "if  -> (empty) else -> return" be converted to just an "if -> return"?